app/resources/cy/translation/common.json

```diff
@@ -21,6 +21,14 @@
   "months": "Ionawr, Chwefror, Mawrth, Ebrill, Mai, Mehefin, Gorffennaf, Awst, Medi, Hydref, Tachwedd, Rhagfyr",
   "supportLinks": "Cysylltiadau Cefnogi",
   "accessibility": "Datganiad Hygyrchedd",
+  "cookieBannerTitle": "TRANSLATE TO WELSH: Cookies on equality and diversity",
```

Comment:
Awaiting one String to be translated to Welsh ("Cookies on equality and diversity") , as able to find Welsh translations for all other new strings from other HMCTS projects.